rtcp-packet_test.go

```diff
@@ -0,0 +1,157 @@
+package rtp
+
+import (
+	"bytes"
+	"encoding/hex"
+	"testing"
+)
+
+
+func unreachable(t *testing.T) {
+  t.Fatalf("unreachable reached")
+}
+
+func assertEqual(t *testing.T, a interface{}, b interface{}) {
+	if a != b {
+    println("actual: ", a)
+    println("expected: ", b)
```

Comment:
Don't use this method.  Use `fmt.Println`, or even better `fmt.Printf("actual: %+v\n", a)`

Reply:
Also, the spacing here indicates that you need to run `gofmt -s`

Reply:
`gofmt -s` seems to completely mess up the code. It looks like it is using 4 tabs instead of 1 tab everywhere